docs/client-guide.adoc

```diff
@@ -74,7 +80,7 @@ certificates_path = "/tmp/sota_certificates" <7>
 <4> The frequency, in seconds, with which the SOTA client should poll the server for updates.
 <5> The location SOTA Client should use for temporary package storage until they are processed by the software loading manager.
 <6> The software loading manager backend to use. Possible values are `deb`, `rpm`, and `off`. If an external software loading manager is in use, this should be set to `off`.
-<7> The certificate authorities SOTA Client trusts. Defaults taken from Mozilla Servo.
+<7> The certificate authorities SOTA Client trusts. The recommended defaults are taken from Mozilla Servo, and the default certificates file can be downloaded from https://raw.githubusercontent.com/advancedtelematic/rvi_sota_client/master/run/sota_certificates[the SOTA client repo]. You may also use your own CA file or the operating system's default trusted certificates file, but ATS recommends using the provided certificate file.
```

Comment:
Let's save that change for a later revision. For the customer implementing this week, I think it will be best to eliminate all possible sources of potential error or confusion.